articles/cosmos-db/how-to-setup-rbac.md

```diff
@@ -397,6 +398,18 @@ const client = new CosmosClient({
 });
 ```
 
+### In Python
+
+The Azure Cosmos DB RBAC is currently supported in the [Python SDK V4.3.0b4](sql-api-sdk-python.md).
+
+```python
+aad_credentials = ClientSecretCredential(
```

Comment:
I saw the section mentioned that the examples show the use of ClientSecretCredential, but in Python we also have the DefaultAzureCredential() - that's beyond the scope of this document right?

Reply:
Is ClientSecretCredential what customers primarily use? If so, we should show that example as part of this to match the other SDK examples.
I see a customer [request ](https://github.com/Azure/azure-sdk-for-python/issues/21663) we had for AAD used ClientSecretCredential

Reply:
Really depends on the use-case. I suggest we stick with ClientSecretCredential for the sake of consistency with the other samples.